src/Collapsible.js

```diff
@@ -175,6 +175,13 @@ class Collapsible extends Component {
           className={triggerClassString.trim()}
           onClick={this.handleTriggerClick}
           style={this.props.triggerStyle && this.props.triggerStyle}
+          onKeyPress={(event) => {
+            const { key } = event;
+              if (key === ' ' || key === 'Enter') {
+                this.handleTriggerClick(event);
+              }
+            }}
+            tabIndex={this.props.tabIndex}
```

Comment:
and this should be `tabIndex={this.props.tabIndex && this.props.tabIndex}`

Reply:
Yup, Updated this. Thanks for the feedback.